src/view.cpp

```diff
@@ -359,8 +359,16 @@ void view::open_in_browser(const std::string& url) {
 		cmdline.append("'");
 	}
 	stfl::reset();
-	utils::run_interactively(cmdline, "view::open_in_browser");
+	// store the shell return value
```

Comment:
You don't have to write comments that explain basic things like assignment operator. Reserve them for tricky stuff (basically anything that took you a while to think up) and explanations of your motivation (it's easy enough to figure out *what* the code does, but it's much harder to find out *why* it does that).